easybuild/easyconfigs/h/HMMER/HMMER-3.3.2-gompic-2020b.eb

```diff
@@ -0,0 +1,68 @@
+##
+# EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2012-2014 Uni.Lu/LCSB, NTUA
+# Authors::   Nils Christian <nils.christian@uni.lu>,
+#             Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a
+#  component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-94.html
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'HMMER'
+version = '3.3.2'
+
+homepage = 'http://hmmer.org/'
+description = """HMMER is used for searching sequence databases for homologs
+ of protein sequences, and for making protein sequence alignments. It
+ implements methods using probabilistic models called profile hidden Markov
+ models (profile HMMs).  Compared to BLAST, FASTA, and other sequence
+ alignment and database search tools based on older scoring methodology,
+ HMMER aims to be significantly more accurate and more able to detect remote
+ homologs because of the strength of its underlying mathematical models. In the
+ past, this strength came at significant computational expense, but in the new
+ HMMER3 project, HMMER is now essentially as fast as BLAST."""
+
+toolchain = {'name': 'gompic', 'version': '2020b'}
+
+source_urls = [
+    'http://eddylab.org/software/hmmer/',
+    'http://eddylab.org/software/hmmer%(version_major)s/%(version)s/',
+]
+sources = ['hmmer-%(version)s.tar.gz']
```

Comment:
Why not `SOURCELOWER_TAR_GZ` like the existing easyconfigs?

Reply:
I made this ec from https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/h/HMMER/HMMER-3.3.2-gompi-2020b.eb#L37

Switched in d0ebdf9